### PR TITLE
[Elixir] Updated Elixir references.

### PR DESCRIPTION
**Pull request by chriszimmerman**
Macros are a pretty powerful feature of Elixir, so I added a link to what is, at the moment, the best book suited for learning about macros in Elixir.

### CODE REVIEW

File: elixir.html.markdown [hunk 1]
```diff
@@ -411,7 +412,8 @@ self() #=> #PID<0.27.0>
 
 * [Getting started guide](http://elixir-lang.org/getting_started/1.html) from [elixir webpage](http://elixir-lang.org)
 * [Elixir Documentation](http://elixir-lang.org/docs/master/)
-* ["Programming Elixir"](https://pragprog.com/book/elixir/programming-elixir) by Dave Thomas
+* [Programming Elixir](https://pragprog.com/book/elixir/programming-elixir) by Dave Thomas
```
Comment (dskecse) [inline elixir.html.markdown]: Why did you remove double quotes? This is a book and it should be rendered as `"Some Book" by Author` rather than `Some Book by Author`.

File: elixir.html.markdown [hunk 2]
```diff
@@ -411,7 +412,8 @@ self() #=> #PID<0.27.0>
...
 * [Elixir Cheat Sheet](http://media.pragprog.com/titles/elixir/ElixirCheat.pdf)
-* ["Learn You Some Erlang for Great Good!"](http://learnyousomeerlang.com/) by Fred Hebert
-* ["Programming Erlang: Software for a Concurrent World"](https://pragprog.com/book/jaerlang2/programming-erlang) by Joe Armstrong
+* [Learn You Some Erlang for Great Good!](http://learnyousomeerlang.com/) by Fred Hebert
```
Comment (dskecse) [inline elixir.html.markdown]: Same here.

File: elixir.html.markdown (hunk 1 above)
Comment (chriszimmerman) [inline elixir.html.markdown]: I made the (probably poor) assumption that since the title of the book is in a link, it's pretty easy to see what the title of the book is. If this is a convention that I am violating, I will gladly change it.

Comment (vendethiel) [inline elixir.html.markdown]: I think you followed what every tutorial on the website does, actually.

**Comment by vendethiel** [comment]
however, I'm afraid we reserve the "contributors" tag to significant contributions

**Comment by chriszimmerman** [comment]
@vendethiel Updated.

**Comment by dskecse** [comment]
@chriszimmerman I'd leave the other two items untouched.

**Comment by chriszimmerman** [comment]
@dskecse I'm assuming you mean the book entries? Of course I'll leave them in there. :D I was just addressing @vendethiel 's comment.

**Comment by vendethiel** [comment]
thank you :)

**Comment by dskecse** [comment]
@chriszimmerman by leaving those untouched I meant you to bring back deleted double quotes.

**Comment by vendethiel** [comment]
Paid resources are a no-no, so I'm going to decline this PR
